.devcontainer/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.12-slim
+
+# Create directory for the application
```

Comment:
We can consider setting a non-root user and then later switching back
```
# Create a non-root user
RUN useradd -m appuser

# Create directory for the application and set permissions
RUN mkdir /app && chown appuser:appuser /app
```

and later on 
```
# Switch to the non-root user
USER appuser
```

Reply:
Good suggestion, I added this.